root/tag/NotFound.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2018 Shamroy Pellew
+//
+// This file is part of MusicBrainz, the open internet music database,
+// and is licensed under the GPL version 2, or (at your option) any
+// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+
+const React = require('react');
+const NotFound = require('../components/NotFound');
+const {l} = require('../static/scripts/common/i18n');
+
+const TagNotFound = () => (
+  <NotFound title={l('Tag Not Used')}>
+    <p>{l('No MusicBrainz entities have yet been tagged with "' + $c.stash.tag + '".')}</p>
+    <p>{l('If you wish to use this tag, please {search_url|search} for the entity first and apply the tag using the sidebar.',
```

Comment:
Please don't change the strings unless necessary. I saw you changed `{url|search}` to `{search_url|search}` and had to go through all the rest to make sure there weren't other changes. These changes require translators to retranslate things for no reason.

(No need to push fixes, I'm taking care of it.)